snap/snapcraft.yaml

```diff
@@ -242,6 +246,9 @@ parts:
       - usr/lib/*/libgbm*.so.*
       - usr/lib/*/libglapi*.so.*
       - usr/lib/*/libgraphite2*.so.*
+      - usr/lib/*/libgstapp-1.0*.so.*
+      - usr/lib/*/libgstbase-1.0*.so.*
+      - usr/lib/*/libgstreamer-1.0*.so.*
```

Comment:
myself having no idea what's the correct way to deploy gstreamer, just double-checking if you have tried building the snap, installing into a live session, and tried if it plays the sound ok?

Reply:
I'll try and check

Reply:
Doesn't work yet
```
Apr 13 11:48:50 ubuntu ubuntu-desktop-installer_ubuntu-desktop-installer.desktop[5654]: flutter: DEBUG subiquity_client: POST http://localhost/meta/mark_configured?endpoint_names=%5B%22mirror%22%2C%22proxy%22%2C%22ssh%22%2C%22snaplist%22%2C%22ubuntu_pro%22%5D
Apr 13 11:48:50 ubuntu ubuntu-desktop-installer_ubuntu-desktop-installer.desktop[5654]: flutter: DEBUG subiquity_client: ==> markConfigured([mirror, proxy, ssh, snaplist, ubuntu_pro]) null
Apr 13 11:48:50 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstadaptivedemux2.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstadaptivedemux2.so: undefined symbol: gst_event_type_to_sticky_ordering
Apr 13 11:48:51 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstcoreelements.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstcoreelements.so: undefined symbol: gst_debug_log_id
Apr 13 11:48:51 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstnavigationtest.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstnavigationtest.so: undefined symbol: gst_navigation_event_parse_touch_event
Apr 13 11:48:51 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstopengl.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstopengl.so: undefined symbol: gst_navigation_event_new_mouse_scroll
Apr 13 11:48:51 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstplayback.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstplayback.so: undefined symbol: gst_query_parse_selectable
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideo4linux2.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideo4linux2.so: undefined symbol: gst_video_format_info_extrapolate_stride
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideobox.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideobox.so: undefined symbol: gst_navigation_event_set_coordinates
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideoconvertscale.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideoconvertscale.so: undefined symbol: gst_navigation_event_set_coordinates
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideocrop.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideocrop.so: undefined symbol: gst_navigation_event_set_coordinates
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideofilter.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstvideofilter.so: undefined symbol: gst_navigation_event_set_coordinates
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstwavparse.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstwavparse.so: undefined symbol: gst_type_find_helper_for_buffer_with_caps
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstximagesink.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstximagesink.so: undefined symbol: gst_navigation_event_new_touch_up
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstxvimagesink.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstxvimagesink.so: undefined symbol: gst_navigation_event_new_touch_up
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libcluttergst3.so': /lib/x86_64-linux-gnu/libclutter-1.0.so.0: undefined symbol: g_string_free_and_steal
Apr 13 11:48:52 ubuntu gst-plugin-scan[5685]: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstgtk.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstgtk.so: undefined symbol: gst_navigation_event_new_mouse_scroll
Apr 13 11:48:52 ubuntu ubuntu-desktop-installer_ubuntu-desktop-installer.desktop[5654]: terminate called after throwing an instance of 'char const*'
Apr 13 11:48:53 ubuntu systemd[1346]: app-gnome-ubuntu\x2ddesktop\x2dinstaller_ubuntu\x2ddesktop\x2dinstaller-5654.scope: Consumed 4.562s CPU time.
```

Priming `usr/lib/*/gstreamer-1.0/*` should do the trick